kvbc/src/Replica.cpp

```diff
@@ -130,6 +130,52 @@ void Replica::registerReconfigurationHandlers(std::shared_ptr<bftEngine::IReques
   stReconfigurationSM_->pruneOnStartup();
 }
 
+void Replica::handleWedgeEvent() {
+  auto lastExecutedSeqNum = m_replicaPtr->getLastExecutedSequenceNum();
+  if (lastExecutedSeqNum == 0) return;
+  uint64_t wedgeBlock{0};
+  auto version = getLatestVersion(kConcordInternalCategoryId, std::string{keyTypes::reconfiguration_wedge_key});
+  if (!version.has_value()) return;
+  wedgeBlock = version.value().version;
+
+  uint64_t wedgeBftSeqNum{0};
+  const auto &sn_val = get(kConcordInternalCategoryId, std::string{keyTypes::bft_seq_num_key}, wedgeBlock);
```

Comment:
Consider making a getter function which will hide the impl details of getting a reconfig data.
There's a lot of code duplication.

Reply:
done